src/main/java/seedu/trackermon/logic/parser/AddCommandParser.java

```diff
@@ -37,10 +39,11 @@ public AddCommand parse(String args) throws ParseException {
 
         Name name = ParserUtil.parseName(argMultimap.getValue(PREFIX_NAME).get());
         Status status = ParserUtil.parseStatus(argMultimap.getValue(PREFIX_STATUS).get());
+        Rating rating = ParserUtil.parseRating(argMultimap.getValue(PREFIX_RATING).orElse("0"));
```

Comment:
Maybe a default name constant can be used instead of "0", to avoid magic literal.